### PR TITLE
JSONSampler Integration

### PR DESCRIPTION
**Pull request by nericksen**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Adding the JSONSampler integration.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - arguments:
+    - description: The key to change.  Must also set value arguement.  Can be comma
+        separated to change multiple values.
+      name: key
+    - description: The new key value. Must also set key argument. Can be comma separated
+        to support changing multiple values.
+      name: value
+    description: Sample the provided JSON and return the results to the Context and
+      Warroom.  Can use key and value arg to change a JSON values if desired.
+    name: sampler
```
Comment (Itay4) [inline Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml:42]: in order to not be too generic, lets modify the command name to be more specific with the integration name as prefix and `-` as separator

File: Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  required: true
+  type: 12
+- display: The incident name to give to the created incident
+  name: name
+  required: false
+  type: 0
+description: |-
+  A utility for testing incident fetching with mock JSON data.
+display: JSONSampler
+name: JSONSampler
```
Comment (Itay4) [inline Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml:30]: i'm thinking if we can improve the name here a bit
we already have an integration named `Sample Incident Generator` so perhaps we should go with `JSON Sample Incident Generator`
what do you think?

**Review by Itay4 — COMMENTED** [review]
cool ! 👍 
i've added 2 comments regarding the naming which bothers me a bit
let me know what you think

File: Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml (hunk 1 above)
Comment (nericksen) [reply Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml]: Updated this to be more in line with other integrations as suggested.

File: Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml (hunk 2 above)
Comment (nericksen) [reply Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml]: That sounds good to me! I have updated the files accordingly.

**Review by Itay4 — APPROVED** [review]
looks good !

**Pull request merged**